tutoraspects/plugin.py

```diff
@@ -72,9 +72,20 @@
         ######################
         # ClickHouse Settings
         ("CLICKHOUSE_HOST", "clickhouse"),
-        ("CLICKHOUSE_PORT", "9000"),
+        ("CLICKHOUSE_CLIENT_HTTP_PORT", "9000"),
+        ("CLICKHOUSE_CLIENT_HTTPS_PORT", "9440"),
+        (
+            "CLICKHOUSE_CLIENT_PORT",
+            "{% if CLICKHOUSE_SECURE_CONNECTION %}"
+            "{{CLICKHOUSE_CLIENT_HTTPS_PORT}}{% else %}{{CLICKHOUSE_CLIENT_HTTP_PORT}}{% endif %}",
+        ),
         ("CLICKHOUSE_HTTP_PORT", "8123"),
         ("CLICKHOUSE_HTTPS_PORT", "8443"),
+        (
+            "CLICKHOUSE_PORT",
```

Comment:
This didn't work for me. Setting `CLICKHOUSE_SECURE_CONNECTION` to `true` did not change the `CLICKHOUSE_PORT`. Any idea if it's something I did wrong?

Reply:
Ah, this needs to be changed to `CLICKHOUSE_PORT`: https://github.com/openedx/tutor-contrib-aspects/blob/8725bf551ef562760c56ceef692a014192b0a850/tutoraspects/templates/aspects/apps/scripts/aspects/profiles.yml#L8

Reply:
Same here: https://github.com/openedx/tutor-contrib-aspects/blob/d8ff56c7cf6f46b6e2fe756a7d8c518dad096853/tutoraspects/templates/aspects/apps/scripts/aspects/clickhouse-demo-xapi-data.sh#L11

Reply:
The config default tutor settings, will not update in your config.yml. The values in the config.yaml will override the default ones.

I will update those references